skaffold.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: skaffold/v2beta7
+kind: Config
+build:
+  tagPolicy:
+    sha256: {}
+  artifacts:
+  - image: rayyan-store
+    context: server
+    docker:
+      dockerfile: Dockerfile-dev
+deploy:
+  helm:
+    releases:
+    - name: rayyan-store
+      namespace: rayyan-dev
+      chartPath: server/helm/rayyan-store
+      artifactOverrides:
+        rayyanStore.image: rayyan-store
+      wait: true
+      # Manually run `helm dep update` inside chartPath every time dependencies are changed in Chart.yaml
+      skipBuildDependencies: true
+      packaged: {}
+      valuesFiles:
+        - server/helm/rayyan-store/value-overrides/local.yaml
+profiles:
+- name: ci
+  patches:
+  - op: replace
+    path: /build/tagPolicy
+    value:
+      envTemplate:
+        template: "{{.BUILD_BUILDID}}"
+  - op: replace
+    path: /deploy/helm/releases/0/packaged
+    value:
+      version: 1.0.{{.BUILD_BUILDID}}
+  - op: replace
+    path: /build/artifacts/0/docker
+    value:
+      dockerfile: Dockerfile
```

```suggestion
    value: Dockerfile
```
This is more explicit than replacing the whole block (we may add other options later).